modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/OrganizationResourceTest.java

```diff
@@ -762,6 +792,16 @@ protected Organization testPutOrganization_addOrganization()
 			randomOrganization());
 	}
 
+	private AccountEntry _addAccountEntry() throws Exception {
+		return _accountEntryLocalService.addAccountEntry(
```

Comment:
because it's just a "single line" can we inline this in the two places we are creating the accountEntry?